### ISSUE TITLE
BitMart 0.06% trading fee for CCXT users!

### ISSUE THREAD

**Issue by kroitor** [ccxt/ccxt#7443]
### Exclusive Deal for CCXT Developers/Users:

Register & sign up to enjoy 0.06% trading fee!

- https://www.bitmart.com/register/en?r=rQCFLh

[![Register & sign up to enjoy 0.06% trading fee!](https://user-images.githubusercontent.com/1294454/89838001-c66cdf80-db72-11ea-8f23-da9c544828d5.png)](https://www.bitmart.com/register/en?r=rQCFLh)

**Comment by cryptoAcc0unt** [comment]
Bitmart has recently started freezing a bunch of users, I'm worried they are the latest exit scam. CCXT might want to consider delisting them

**Comment by silert** [comment]
@cryptoAcc0unt Recently they have published new trading rules: https://bitmart.zendesk.com/hc/en-us/articles/360052757433

For example "High-frequency trading, illegal quantitative trading, or arbitrage trading" are prohibited.

**Comment by cryptoAcc0unt** [comment]
@silert yeah its the same approach all these shady exchanges use to justify freezing accounts en masse. I wasn't doing anything to violate the new rules

**Comment by kroitor** [comment]
@silert 

> High-frequency trading, illegal quantitative trading, or arbitrage trading

Can you please provide definitions for those terms? 

Namely, what is considered _High-frequency trading_, what is considered _illegal quantitative trading_. 

On the arbitrage trading – it's uncommon to prohibit this activity, since market-making and arbitrage are closely tied together. 

Just asking these questions, as we had a similar investigation here: https://github.com/ccxt/ccxt/issues/7066. In the end, the exchange has unlocked the users' funds because there was no strict definition of the exact violated trading limitations, and the exchange did not provide the exact reason for fundlocking:

In other words, if the arbitrage trading or high-frequency trading is not allowed without an exact definition of rules and possible violations – that's essentially the same as just saying _"Welcome to our exchange, allow everything except the trading itself"_.

**Comment by silert** [comment]
@kroitor I'm not represent the Bitmart exchange, I just found the link with new rules and I also had difficulties with withdrawals on Bitmart in the past.

**Comment by kroitor** [comment]
@silert 
> I'm not represent the Bitmart exchange
Yeah, pardon, i've misread it at first, didn't mean that. Thanks for sharing! We will send these questions to BitMart.

**Comment by Rekamserg** [comment]
Good day.
I am one of those whose withdrawals from the exchange are blocked after the introduction of their new rules. From the whole list of rules, I did not break anything. I created a ticket, but I received no comment from the exchange side.

Not so long ago, I received a reply from administrators in telegrams that these locks are risk control and soon all functions of the account will resume. I have been working on this exchange for over a year and everything was fine with me. However, their support is slow. I hope that in this case, their team will answer all blocked users and resolve their questions.

**Comment by Rekamserg** [comment]
Good day. An hour ago I tried to withdraw coins. Everything went well and I received my funds.
One note: the exchange team needs to comment on the situation. So that users do not worry about their funds.
